hooks/post_gen_project.sh

```diff
@@ -41,6 +41,17 @@ fi
 
 {% endif %}
 
+
+{% if cookiecutter.use_wagtail == 'n' %}
+# Delete wagtail if not using
+echo "Deleting wagtail app";
```

Comment:
I want to figure out a better way to do this - One option would be including the files and just having them empty